bittensor/_axon/__init__.py

```diff
@@ -54,6 +54,7 @@ def __new__(
             synapse_last_hidden: 'Callable' = None,
             synapse_causal_lm: 'Callable' = None,
             synapse_seq_2_seq: 'Callable' = None,
+            synapse_checks: 'Callable' = None,
```

Comment:
nit: we can use straight up `typing.Callable` in our type annotations since we're pinned to Python >= 3.5 now.

Reply:
Good point; we actually use that quite a bit. Let's add it as a ticket, we should change all of the 'Callable' to proper typing